src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster-review.component.ts

```diff
@@ -15,15 +17,21 @@ import { InventoryDevice } from '../inventory/inventory-devices/inventory-device
 export class CreateClusterReviewComponent implements OnInit {
   hosts: object[] = [];
   hostsDetails: object;
-  hostsByLabel: object;
+  hostsByService: object;
   hostsCount: number;
-  labelOccurrences = {};
-  hostsCountPerLabel: object[] = [];
-  uniqueLabels: Set<string> = new Set();
+  serviceCount: number;
+  serviceOccurrences = {};
+  hostsCountPerService: object[] = [];
+  uniqueServices: Set<string> = new Set();
   filteredDevices: InventoryDevice[] = [];
   capacity = 0;
+  services: Array<CephServiceSpec> = [];
 
-  constructor(private hostService: HostService, public wizardStepService: WizardStepsService) {}
+  constructor(
+    public wizardStepService: WizardStepsService,
```

Comment:
Shouldn't they be called exactly the same (except for the first upper-case letter)?
```suggestion
    public wizardStepsService: WizardStepsService,
```

Reply:
Done. Thanks